src/App.js

```diff
@@ -1,26 +1,44 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import TodoList from './Components/Todo_list/Todo_list'
```

Comment:
```suggestion
import TodoList from './components/todo/List'
```
Tên thư mục bắt đầu bằng chữ in thường, như quy tắc đặt tên biến (camel case)
Tên Component theo quy tắc đặt tên `Class` theo quy tắc camel case
Sửa lại link import, tên thư mục, và tên Component
Hiện đang đặt tên ko theo quy tắc gì cả, và rất dài dòng